tests/PHPStan/Parser/TypeParserTest.php

```diff
@@ -668,6 +668,24 @@ public function provideParseData(): array
 					Lexer::TOKEN_CLOSE_CURLY_BRACKET
 				),
 			],
+			[
+				'list{
+				 	int,
+				 	string
+				 }',
+				new ArrayShapeNode([
```

Comment:
Would be nice to know the array shape "kind" (array or list) at the ArrayShapeNode class.